os/board/rtl8730e/src/rtl8730e_boot.c

```diff
@@ -273,8 +273,8 @@ void amebasmart_mount_partitions(void)
 }
 
 #ifdef CONFIG_FTL_ENABLED
-extern const u8 ftl_phy_page_num;
-extern const u32 ftl_phy_page_start_addr;
+extern u8 ftl_phy_page_num;
+extern u32 ftl_phy_page_start_addr;
```

Comment:
Could you let me know why this is necessary?

Reply:
The ftl physical page number and start addresses are updated by some line of code (Eg `board/rtl8730e/src/component/usrcfg/rtl8730e/ameba_flashcfg.c`) and should not be declared as constant variable types.